spec/support/shared_examples/busca_logotipo.rb

```diff
@@ -0,0 +1,14 @@
+shared_examples_for 'busca_logotipo' do
+  it 'para layout padrão' do
+    boleto_novo = described_class.new
+    expect(File.exist?(boleto_novo.logotipo)).to be_truthy
+    expect(File.stat(boleto_novo.logotipo).zero?).to be_falsey
+  end
+
+  it 'para layout de carnê' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.